### PR TITLE
Add Fantom FTM and Celo CELO — Add token after fiat on ramp order.

### PR DESCRIPTION
**Pull request by wachunei**
> ⚠️  This PR is branched off https://github.com/MetaMask/metamask-mobile/pull/3768

This PR adds the asset to the token list after completing a Transak order

### CODE REVIEW

File: app/components/UI/FiatOrders/TransakWebView/index.js [hunk 1]
```diff
@@ -47,9 +55,26 @@ class TransakWebView extends PureComponent {
...
+		}
+	};
+
 	handleNavigationStateChange = async (navState) => {
 		if (navState.url.indexOf(AppConstants.FIAT_ORDERS.TRANSAK_REDIRECT_URL) > -1) {
 			const order = handleTransakRedirect(navState.url, this.props.network);
+			this.addTokenToAssetsController(order.cryptocurrency, this.props.network);
 			this.props.addOrder(order);
 			this.props.protectWalletModalVisible();
 			this.props.navigation.dangerouslyGetParent()?.pop();
```
Comment (gantunesr) [inline app/components/UI/FiatOrders/TransakWebView/index.js:80]: At this point the order is already confirmed?

**Comment by gantunesr** [comment]
>  > ⚠️ This PR is branched off https://github.com/MetaMask/metamask-mobile/pull/3768

> This PR adds the asset to the token list after completing a Transak order

Is this PR blocked by #3768?

File: app/components/UI/FiatOrders/TransakWebView/index.js [hunk 2]
```diff
@@ -47,9 +55,26 @@ class TransakWebView extends PureComponent {
 		route: PropTypes.object,
 	};
 
+	addTokenToAssetsController = async (symbol, chainId) => {
```
[the remark this answers is not in the document]
Comment (wachunei) [reply app/components/UI/FiatOrders/TransakWebView/index.js:58]: I will rename

File: app/components/UI/FiatOrders/TransakWebView/index.js (hunk 1 above)
Comment (wachunei) [reply app/components/UI/FiatOrders/TransakWebView/index.js:80]: No, every order is added as pending.

**Comment by wachunei** [comment]
Is not blocked but if we merge this we merge #3768 because it includes its changes.

File: app/constants/on-ramp.js [hunk 3]
```diff
@@ -44,4 +44,50 @@ export const NETWORKS_CHAIN_ID = {
...
+	[NETWORKS_CHAIN_ID.BSC]: 'BNB',
+	[NETWORKS_CHAIN_ID.POLYGON]: 'MATIC',
+	[NETWORKS_CHAIN_ID.AVAXCCHAIN]: 'AVAX',
+	[NETWORKS_CHAIN_ID.CELO]: 'CELO',
+	[NETWORKS_CHAIN_ID.FANTOM]: 'FTM',
+};
+
+export const NETWORK_ALLOWED_TOKENS = {
+	[NETWORKS_CHAIN_ID.MAINNET]: [
+		{ symbol: 'USDT', address: '0xdac17f958d2ee523a2206206994597c13d831ec7', decimals: 6 },
```
Comment (andreahaku) [inline app/constants/on-ramp.js:71]: what about creating constants for all those addresses?

File: app/constants/on-ramp.js [hunk 4]
```diff
@@ -44,4 +44,50 @@ export const NETWORKS_CHAIN_ID = {
...
+	],
+	[NETWORKS_CHAIN_ID.BSC]: [{ symbol: 'BUSD', address: '0x4fabb145d64652a948d72533023f6e7a623c7c53', decimals: 18 }],
+	[NETWORKS_CHAIN_ID.POLYGON]: [
+		{ symbol: 'USDT', address: '0xc2132d05d31c914a87c6611c10748aeb04b58e8f', decimals: 6 },
+		{ symbol: 'USDC', address: '0x2791bca1f2de4661ed88a30c99a7a9449aa84174', decimals: 6 },
+		{ symbol: 'DAI', address: '0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063', decimals: 18 },
+	],
+};
+
+export const NETWORK_PARAMETERS = Object.keys(NETWORK_NAMES).reduce(
```
Comment (andreahaku) [inline app/constants/on-ramp.js:83]: this is clever ;)

File: app/constants/on-ramp.js (hunk 3 above)
Comment (wachunei) [reply app/constants/on-ramp.js:71]: Since all of this logic will be replaced when we launch the aggregator I don't think is worth to make it a constant.

**Review by gantunesr — APPROVED** [review]

**Pull request merged**